CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to prometheus-sql
+
+**First:** if you're unsure or afraid of _anything_, just ask or submit the issue or pull request anyways. You won't be yelled at for
+giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions,
+and don't want a wall of rules to get in the way of that.
+
+However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover
+what we're looking for. By addressing all the points we're looking for, it raises the chances we can quickly merge or address your contributions.
+
```

Comment:
Out of curiosity did you break these lines for the easy of editing the markdown? (as supposed to having one big block of text)